mathics/builtin/numbers/integer.py

```diff
@@ -88,6 +92,8 @@ def apply(self, n, evaluation):
 
 class Ceiling(SympyFunction):
```

Comment:
Not related to this PR, just me curious: do you know why WL have Ceiling and Floor, instead of Ceil and Floor, or Ceiling and Flooring?

Reply:
Good question!